vllm/utils/__init__.py

```diff
@@ -3423,3 +3423,22 @@ def decorate_logs(process_name: Optional[str] = None) -> None:
     pid = os.getpid()
     _add_prefix(sys.stdout, process_name, pid)
     _add_prefix(sys.stderr, process_name, pid)
+
+
+def is_port_available(host: str = "127.0.0.1", port: int = 8000) -> bool:
```

Comment:
Can you check if thee is an existing function for this

Reply:
I did not find any similar functions after a preliminary check.